.github/workflows/upload-idf-component.yml

```diff
@@ -0,0 +1,50 @@
+name: Push components to https://components.espressif.com
```

Comment:
could we set a shorter name that fits in the github view ?
<img width="343" alt="image" src="https://github.com/user-attachments/assets/9c399bed-912a-4dc2-ad25-8e0788a07197" />

Reply:
sure